Libraries/Components/Touchable/TouchableHighlight.js

```diff
@@ -225,7 +275,7 @@ var TouchableHighlight = React.createClass({
 
   render: function() {
     return (
-      <View
+      <TVView
```

Comment:
Feel free to add that to Platform.ios.js if that's the best way forward. I would still prefer the original approach I commented here (unless you can tell me why it doesn't work of course ;) )